### PR TITLE
Add +draft/language client tag.

### PR DESCRIPTION
**Pull request by ProgVal**
A small specification for once, hopefully without too many implications. :)

Some use cases I can think of:

* Using specific fonts for different languages, like browsers do. (Useful when the default font is partial, or [when the same unicode codepoint has variants depending on language](https://chinese.stackexchange.com/questions/792/characters-which-have-several-different-shapes))
* Helping text to speech software
* Bots which provide automatic translation don't need to guess the language (which usually involves an HTTP API call); Limnoria has a plugin doing that.
* You can add a button next to languages in a foreign language, like "Translate with Bing" / "Translate with Google" on some microblogging websites
* Hiding/folding messages in some languages if the user so desires, eg. for multilingual channels

### CODE REVIEW

File: client-tags/language.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    C: @+draft/language=fra PRIVMSG #channel :Je parle aussi français.
+
+### Multiline message
+
+NOTE: In this examples, `<SPACE>` indicates a space character which would
+otherwise not be clearly visible.
+
+    C: BATCH +123 draft/multiline #channel
+    C: @batch=123 +draft/language=eng PRIVMSG #channel :The French word for "cat" is<SPACE>
+    C: @batch=123;draft/multiline-concat;+draft/language=fra +draft/language=fra PRIVMSG #channel :"chat"
```
Comment (DarthGandalf) [inline client-tags/language.md:86]: Why language is repeated, and second one as a separate parameter?

File: client-tags/language.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+
+### Dependencies
+
+Clients wishing to use this tag MUST negotiate the
+[`message-tags`](../extensions/message-tags.html) capability with the server.
+
+### Format
+
+The language tag is sent by a client with the client-only prefix `+` and
+its value must be a a ISO 639-3 code.
```
Comment (DarthGandalf) [inline client-tags/language.md:38]: No difference between e.g. British and American English?

File: client-tags/language.md (hunk 1 above)
Comment (ProgVal) [reply client-tags/language.md:86]: fixed

File: client-tags/language.md (hunk 2 above)
Comment (ProgVal) [reply client-tags/language.md:38]: No. Even variants of arabics are considered the same macrolanguage.

Comment (prawnsalad) [reply client-tags/language.md:38]: There's https://tools.ietf.org/html/bcp47 which is the commonly used `en-US` `en-GB` `en` `fr` etc

**Comment by ProgVal** [comment]
There doesn't seem to be much interest in this, so I'm closing the proposal. Ping me if you want to reopen it.

**Pull request closed without merging**